src/main/java/unicash/logic/UniCashMessages.java

```diff
@@ -21,6 +31,31 @@ public class UniCashMessages {
             "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_INVALID_MONTH = "Month must be between 1 and 12 (inclusive).";
 
+    public static final String MESSAGE_UNICASH_WELCOME = "Welcome to UniCa$h!"
+            + "\n\n"
+            + "Available Commands:"
+            + "\n"
+            + AddTransactionCommand.COMMAND_WORD
+            + "\n"
+            + DeleteTransactionCommand.COMMAND_WORD
+            + "\n"
+            + EditTransactionCommand.COMMAND_WORD
+            + "\n"
+            + ListCommand.COMMAND_WORD
+            + "\n"
+            + FindCommand.COMMAND_WORD
+            + "\n"
+            + GetTotalExpenditureCommand.COMMAND_WORD
+            + "\n\n"
```

Comment:
Any reason why the newlines are inconsistent? Also, I'm thinking we should really have delegating this job to StringBuilder, this is very hard to read